src/components/Modal/IdentityConnectModal/AccountsConnectModal.tsx

```diff
@@ -388,9 +415,29 @@ function AccountsConnectModal({ open, onClose }: ModalProps & { onClose: () => v
   )
   const currentType = modalState.currentType
 
+  const queryClient = useQueryClient()
+
+  const handleOnClose = () => {
+    resetValidation()
+    setIsTimerActive(false)
+    resetState(currentAccount)
+    setCurrentType(ModalType.ChooseAccount)
+    queryClient.invalidateQueries({
+      queryKey: ['isProfileValidated', address],
```

Comment:
I'm starting to think we could have a `queryKeys.ts` file for all query keys (to avoid repetition and possible typos)

Reply:
Yes! we should do it in another PR